Source/Scene/Scene.js

```diff
@@ -709,6 +719,37 @@ define([
         this._cameraVR = undefined;
         this._aspectRatioVR = undefined;
 
+        /**
+         * When <code>true</code>, rendering a frame will only occur when needed as determined by changes within the scene.
```

Comment:
Please provide more detail here just like you did in the first comment in this pull request.  In particular, make sure that the reader understands that they will need to explicitly call `requestRender` under many cases.